scripts/get-suppliers-with-unsigned-framework-agreements.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""
+For performance testing, we need suppliers who have completed their application but have not signed the
+framework agreement.
+Get up to 500 suitable suppliers who meet these requirements based on the information in the declaration schema.
+This will be output to CSV file in the specified directory.
+
+Usage:
+    scripts/get-suppliers-with-unsigned-framework-agreements.py <stage> <framework_slug> <output_dir> [options]
+
+Options:
+    --limit=LIMIT       Limit the number of suppliers to find [default: 500]
+"""
+import sys
+
+from dmapiclient import DataAPIClient
+from dmutils.env_helpers import get_api_endpoint_from_stage
+from docopt import docopt
+from collections import Counter
+import os
+import csv
+
+sys.path.insert(0, ".")
+
+from dmscripts.helpers.auth_helpers import get_auth_token
+from dmscripts.helpers.updated_by_helpers import get_user
+
+
+DEFAULT_PASSWORD = "Password1234"
+
+
+def _assess_draft_services(framework_slug, supplier_id,):
+    # A supplier must have at least 1 submitted service
+    counter = Counter()
+
+    for draft_service in data_api_client.find_draft_services_by_framework_iter(framework_slug, supplier_id=supplier_id):
+        counter[draft_service["status"]] += 1
+
+    return counter
+
+
+if __name__ == "__main__":
+    arguments = docopt(__doc__)
+    stage = arguments['<stage>']
+    framework_slug = arguments['<framework_slug>']
+    OUTPUT_DIR = arguments['<output_dir>']
+    SUPPLIER_LIMIT = int(arguments['--limit'])
+
+    if stage == "production":
+        raise Exception("This script is not safe to run in production")
```

Comment:
Is this true for this script?

Reply:
Yes, as this will be collecting supplier accounts, we wont want/need a list of supplier emails